examples/foundational/07d-interruptible-elevenlabs.py

```diff
@@ -79,7 +80,7 @@ async def main():
                 allow_interruptions=True,
                 enable_metrics=True,
                 enable_usage_metrics=True,
-                report_only_initial_ttfb=True,
+                # report_only_initial_ttfb=True,
```

Comment:
we probably don't need any of the changes in this file. `eleven_flash_v2_5` is already the default now.

Reply:
Reverted this file.